fe/fe-core/src/main/java/org/apache/doris/tablefunction/ParquetMetadataTableValuedFunction.java

```diff
@@ -0,0 +1,373 @@
+            new Column("physical_type", PrimitiveType.STRING, true),
+            new Column("logical_type", PrimitiveType.STRING, true),
+            new Column("repetition_level", PrimitiveType.INT, true),
+            new Column("definition_level", PrimitiveType.INT, true),
+            new Column("type_length", PrimitiveType.INT, true),
+            new Column("precision", PrimitiveType.INT, true),
+            new Column("scale", PrimitiveType.INT, true),
+            new Column("is_nullable", PrimitiveType.BOOLEAN, true)
+    );
+
+    private static final ImmutableList<Column> PARQUET_METADATA_COLUMNS = ImmutableList.of(
+            new Column("file_name", PrimitiveType.STRING, true),
+            new Column("row_group_id", PrimitiveType.INT, true),
+            new Column("column_id", PrimitiveType.INT, true),
+            new Column("column_name", PrimitiveType.STRING, true),
+            new Column("column_path", PrimitiveType.STRING, true),
+            new Column("physical_type", PrimitiveType.STRING, true),
+            new Column("logical_type", PrimitiveType.STRING, true),
+            new Column("type_length", PrimitiveType.INT, true),
+            new Column("converted_type", PrimitiveType.STRING, true),
+            new Column("num_values", PrimitiveType.BIGINT, true),
+            new Column("null_count", PrimitiveType.BIGINT, true),
+            new Column("distinct_count", PrimitiveType.BIGINT, true),
+            new Column("encodings", PrimitiveType.STRING, true),
+            new Column("compression", PrimitiveType.STRING, true),
+            new Column("data_page_offset", PrimitiveType.BIGINT, true),
+            new Column("index_page_offset", PrimitiveType.BIGINT, true),
+            new Column("dictionary_page_offset", PrimitiveType.BIGINT, true),
+            new Column("total_compressed_size", PrimitiveType.BIGINT, true),
+            new Column("total_uncompressed_size", PrimitiveType.BIGINT, true),
+            new Column("statistics_min", ScalarType.createStringType(), true),
+            new Column("statistics_max", ScalarType.createStringType(), true)
+    );
+
+    private final List<String> paths;
+    private final String mode;
+    // File system info for remote Parquet access (e.g. S3).
+    private final TFileType fileType;
+    private final Map<String, String> properties;
+
+    public ParquetMetadataTableValuedFunction(Map<String, String> params) throws AnalysisException {
+        Map<String, String> normalizedParams = params.entrySet().stream().collect(Collectors.toMap(
+                entry -> entry.getKey().toLowerCase(),
+                Map.Entry::getValue,
+                (value1, value2) -> value2
+        ));
+        String rawPath = normalizedParams.get(PATH);
+        if (Strings.isNullOrEmpty(rawPath)) {
+            throw new AnalysisException("Property 'path' is required for parquet_metadata");
+        }
+        List<String> parsedPaths = Arrays.stream(rawPath.split(","))
```

Comment:
I think we don't need file list with comma, only:
1. full file name
2. file name with wildcard